recipes-tag/eiger-io/meta.yaml

```diff
@@ -1,36 +1,35 @@
-{% set version = "1.0.0" %}
+{% set version = "2.0.0" %}
 package:
-  name: eiger-io
-  version: {{ version }}
+    name: eiger-io
+    version: {{ version }}
 
 source:
-  url: https://github.com/NSLS-II-CHX/eiger-io/archive/v{{ version }}.tar.gz
-  fn: eiger-io-v{{ version }}.tar.gz
-  sha256: 2de8b9fae8f9d9821f3380582e1d3af0213a172fc60086f4859e6b7ff19c3f0e
+    url: https://github.com/NSLS-II-CHX/eiger-io/archive/v{{ version }}.tar.gz
```

Comment:
maybe `git-url: https://github.com/NSLS-II-CHX/eiger-io` and `git-rev: v{{ version }}
` as @licode suggested?
(https://github.com/NSLS-II/lightsource2-recipes/blob/master/recipes-tag/bluesky/meta.yaml#L8)
just to maintain consistency